### ISSUE TITLE
Color specification for each category

### ISSUE THREAD

**Issue by KatsuSofJp** [ArrobeFr/jquery-calendar#1]
Hi,

Is there any way to specify colors to specific category ?

I tried setEventCategoriesColors function, but it's not work well.

Is this bug ?

Also, any other way to fill my purpose?

help, please

**Comment by LeoVillaretArrobe** [comment]
Hi,

Yes, there is a bug, thanks for reporting.

I am working on that now.

**Comment by LeoVillaretArrobe** [comment]
This bug is fixed in [1.0.10](https://github.com/ArrobeFr/jquery-calendar/releases/tag/1.0.10).

It was released on [NPM](https://www.npmjs.com/package/arrobefr-jquery-calendar) too.

Here is a sample code that works now :

```js
var calendar = $('#calendar').Calendar({
        locale: 'en',
        weekday: {
          timeline: {
            intervalMinutes: 30,
            fromHour: 9
          }
        },
        events: [
          {
            start: now.startOf('week').add(10, 'h').format('X'),
            end: now.startOf('week').add(11, 'h').format('X'),
            title: '2',
            content: 'Hello World! <br> <p>Foo Bar</p>',
            category:'Professionnal'
          },
          {
            start: now.startOf('week').add(11, 'h').format('X'),
            end: now.startOf('week').add(12, 'h').format('X'),
            title: '3',
            content: 'Hello World! <br> <p>Foo Bar</p>',
            category:'Personnal'
          }
        ],
        daynotes: []
      });
      calendar.setEventCategoriesColors([
        {
          category: "Professionnal",
          color: "#3F51B5"
        }
      ]);
      calendar.init();
```
*The "Professional" category will always be indigo "#3F51B5".*

Please update to the latest revision and give me a feedback.

**Comment by KatsuSofJp** [comment]
Hi

Thank you for quick reply.

I tryed 1.0.10 you published yesterday, and it works fine !

I appreciate your greate work.
